app/uk/gov/hmrc/agentclientrelationships/controllers/Relationships.scala

```diff
@@ -85,12 +85,13 @@ class Relationships @Inject()(val gg: GovernmentGatewayProxyConnector,
     val sa = a.toSet
 
     if (sa.isEmpty) {
-      Logger.warn("The sa references are empty.")
-      Future.successful(Set.empty)
+      Logger.warn("The sa references in mapping store are empty.")
```

Comment:
the sa refs are from cease not the mapping store (agent mapping service).
perhaps for clarity we should rename the variables also. 'a' and 'b' are confusing, and so is 'sa' as we have sa ids from both cease and agent-mapping.

perhaps rename to something like:
'a' -> 'cesaIds' 
'sa' -> 'cesaIdSet'
'b' -> mappingServiceCall
'x' -> mappingServiceIds